src/components/ui/health-indicator.tsx

```
@@ -0,0 +1,198 @@
+      case 'checking':
+        return (
+          <motion.div
+            animate={{ rotate: 360 }}
+            transition={{ duration: 1, repeat: Infinity, ease: 'linear' }}
+          >
+            <RefreshCw className={`${getIconClasses(size)} ${
+              isDarkMode ? 'text-blue-400' : 'text-blue-600'
+            }`} />
+          </motion.div>
+        );
+      case 'unknown':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)